Huntgroup

amateo opened the pull request:
There was duplicate functionality for huntgroups since #93. Since that PR, freeradius::module::preprocess and freeradius::module::huntgroup wasn't working.
This PR deprecates freeradius::module::preprocess (while still works).

djjudas21 commented:
Can you briefly explain these changes, please? I don't use huntgroups and I didn't write this original code :smile:

amateo commented:
I made a PR #59 in Feb'17. That PR added `freeradius::module::preprocess` to configure preprocess module (basically, to enable the module and initialize an empty huntgroup file), and `freeradius::module::huntgroup` was added too to create huntgroups entries.

Later, PR #93 did the same in a different way. In that PR the empty huntgroup file was created directly in ```init.pp``` file and `freeradius::huntgroup` is added to add huntgroups entries.

The problem is that this later PR conflicts with the former.

What I have done in this new PR is:

- Changed `freeradius::module::preprocess` so that it just enables `preprocess` module (and doesn't create the empty huntgroup file)
- Changed `freeradius::module::huntgroup` to call `freeradius::huntgroup` and display a deprecation warning message.

This way, anyone can continue using huntgroups in the same that #59 or in the #93 way, without any problem.

djjudas21 commented:
OK, sounds good. Thanks!
